src/main/java/com/clubobsidian/dynamicgui/manager/dynamicgui/cooldown/CooldownManager.java

```diff
@@ -0,0 +1,263 @@
+package com.clubobsidian.dynamicgui.manager.dynamicgui.cooldown;
+
+import java.io.File;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import com.clubobsidian.dynamicgui.DynamicGui;
+import com.clubobsidian.dynamicgui.entity.PlayerWrapper;
+import com.clubobsidian.dynamicgui.plugin.DynamicGuiPlugin;
+import com.clubobsidian.dynamicgui.server.FakeServer;
+import com.clubobsidian.wrappy.Configuration;
+import com.clubobsidian.wrappy.ConfigurationSection;
+
+public class CooldownManager {
+
+	private static CooldownManager instance;
+	
+	public static CooldownManager get()
+	{
+		if(instance == null)
+		{
+			instance = new CooldownManager();
+		}
+		
+		return instance;
+	}
+	
+	private Map<UUID, Map<String, Cooldown>> cooldowns;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/virustotalop/DynamicGui/pullRequest?prid=4627182)